vite.config.js

```diff
@@ -6,15 +6,16 @@ import jsxExample from './docs/src/plugins/jsxExample.cjs';
 import propTable from './docs/src/plugins/propTable.cjs';
 import { presetWarp } from '@warp-ds/uno'
 import uno from 'unocss/vite'
-import { classes } from '@warp-ds/component-classes/classes';
+import { classes } from '@warp-ds/css/component-classes/classes';
+
 
 
 export default function config() {
   return {
     // base: '/',
     plugins: [
       uno({
-        presets: [presetWarp({ usePreflight: true })],
+        presets: [presetWarp({ skipResets: true })],
         safelist: classes,
```

Comment:
Don't we need this code? 

```
let reset;
async function getReset() {
  if (reset) return reset;
  else {
    reset = (await fetch('https://assets.finn.no/pkg/@warp-ds/css/1.0.0-alpha.33/resets.min.css')).text();
    return reset;
  }
}

...
```
I see that we are including the reset in storybook, but not in the index.html 🤔

Reply:
oh, I just looked at vue, I guess we want to move the reset to index.html instead of the `.storybook/preview-body.html` file?

Reply:
Or is it not necessary to add it, as we expect the teams to add it?  🤔

Reply:
`index.html` is only used for an old documentation site (the one you see in https://react.fabric-ds.io/), which we haven't yet removed from this repo. It was mentioned at some point that we might want to remove it but we didn't prioritize it. Until we do that, just for the sake of consistency, I added the resets there, too 😁

`.storybook/preview-body.html` should actually be renamed to `preview-head.html` as its contents belong more in the head of the storybook's component page ([storybook docs](https://storybook.js.org/docs/react/configure/story-rendering#adding-to-head)). If we remove the styles from there completely, we wouldn't have access to the stylesheets needed to load stories.

Reply:
Renamed file to `preview-head.html` in [127eb77](https://github.com/warp-ds/react/pull/55/commits/127eb7767dba7b8aa9df9677fdaaeccc35ef9add) and tested that all still looks correct in storybook.